lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -102,6 +102,11 @@ div.course-wrapper {
     @extend .content;
     padding: ($baseline*2);
     line-height: 1.6;
+    .xblock {
+        overflow-wrap: break-word;
+        word-wrap: break-word;
+        word-break: break-all;
```

Comment:
I think this would be safer as 
```
word-break: break-word;
```
so we don't get words breaking all over in every xblock (like the bla-h in your screenshot).